setup.py

```diff
@@ -15,6 +15,7 @@
 
 install_requires = [
     "ply>=3.4,<4.0",
+    "six>=1.15.0"
```

Comment:
I thought we should pin the max version of six, for six 2.0 may have breaking changes. `six~=1.15` is suitable here.
and it's better to have comma and the end of line.

Reply:
Fixed in d87d5cc